build/package/admin.Dockerfile

```diff
@@ -0,0 +1,69 @@
+# Base Go environment
+# -------------------
+FROM golang:1.21-alpine as base
+WORKDIR /hatchet
+
+RUN apk update && apk add --no-cache gcc musl-dev git protoc protobuf-dev
+
+COPY go.mod go.sum ./
```

Comment:
run prefetch immediately after adding go.mod and go.sum to ensure the earliest caching possible

```suggestion
COPY go.mod go.sum ./
RUN go mod download

# prefetch prisma binaries, so that they will be cached and not downloaded on each change
RUN go run github.com/steebchen/prisma-client-go prefetch
```